definitions/region/native_regions_witch_6_0.yml

```diff
@@ -0,0 +1,279 @@
+- WITCH 6.0:
+  - WITCH 6.0|Brazil:
+      iso3_codes: BRA
+  - WITCH 6.0|Canada:
+      iso3_codes:
+      - CAN
+      - SPM
```

Comment:
I'm leaving the following suggestion to exemplify the changes. Another example can be found [here](https://github.com/IAMconsortium/common-definitions/pull/118/files#diff-9ec0600ef893288e9f8c3448c3acde7a37a6d76793b190bea94c57425e0df48a).
```suggestion
      countries: [ Canada, Saint Pierre and Miquelon ]
```

Reply:
implemented in 161fd936ae5f58d3cdf71a5d00e9455e9ea0ea7f

Reply:
Hi Laurent,

The formatting for countries is different, as shown in the [example](https://github.com/IAMconsortium/common-definitions/pull/118/files#diff-9ec0600ef893288e9f8c3448c3acde7a37a6d76793b190bea94c57425e0df48a).
Instead of a hyphenated list, it's instead a comma-separated list in square brackets.